frontend/app/api/questions/[id]/route.ts

```diff
@@ -19,6 +21,38 @@ export async function PUT(req: Request, { params }: { params: { id: number } })
       }
     });
 
+    // if in production, push the question to farcaster
+    if (process.env.NODE_ENV === "production") {
+      const questionerFarcaster = await prisma.socialProfile.findUniqueOrThrow({
```

Comment:
I'm not sure if `findUniqueOrThrow` is correct here. We don't want to stop a reply from going through just because a person doesn't have a social profile of the type farcaster right? (this pattern happens in other places as well)